bindings/python/pmix.pxi

```diff
@@ -1274,10 +1274,7 @@ cdef dict pmix_unload_value(const pmix_value_t *value):
         pyns = str(value[0].data.proc[0].nspace)
         return {'value':{'nspace':pyns, 'rank':value[0].data.proc[0].rank}, 'val_type':PMIX_PROC}
     elif PMIX_BYTE_OBJECT == value[0].type:
-        mybytes = <char*> PyMem_Malloc(value[0].data.bo.size)
-        if not mybytes:
-            return PMIX_ERR_NOMEM
-        memcpy(mybytes, value[0].data.bo.bytes, value[0].data.bo.size)
+        mybytes = <bytes>value[0].data.bo.bytes[:value[0].data.bo.size]
```

Comment:
Is there an issue here with persistence? The bytes in the `value` struct are not guaranteed to remain upon return from a function call - will this automatically "copy" them?

Reply:
The Cython documentation says that this creates a new bytes object with a copy of the C string, https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#type-casting

Reply:
Perfect! Thx.